config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
I have no problem with the changes made here.

However, I am concerned about whether or not this is the right approach.
Do we know what is in the session causing us to exceed the current 1MB max?  It may be easy to find and remove.
I was hoping that memcached could be the first part of the appliance that could be extracted to an independent container.  This level of configuration makes that more difficult.